reqs/management/commands/link_agencies.py

```diff
@@ -0,0 +1,90 @@
+from functools import singledispatch
+from typing import List, Union
+
+import reversion
+from django.core.management.base import BaseCommand
+
+from reqs.models import Agency, AgencyGroup, Requirement
+
+
+class Command(BaseCommand):
+    help = ("Associate agencies and agency groups with requirements "  # noqa
+            "based on impacted entity field.")
+
+    def handle(self, *args, **options) -> None:
```

Comment:
I was using them as comments (although also running them through a rogue mypy locally) as an alternative to docstring comments about what types a function expects.

I didn't want to lay out a policy for the project, but I do think that regardless of whether or not we expect total mypy coverage, it's better to have type annotations on Python 3 projects than not to have them.